src/index.js

```diff
@@ -1,5 +1 @@
-import printMe from './print.js';
 import './styles.css';
```

Comment:
Hey @Javivy, the method you're using to render todos is not what was requested of you to do. You are currently loading data from your dist folder that is compiled by Webpack but that is not what the project required of you.

**See the project requirements and reflect on the points I made below**
![image](https://user-images.githubusercontent.com/50941074/192903792-82b63ade-e920-4062-b0ea-6fdb4559f17c.png)

Kindly take note of the following:

- [ ] You are to have an array with to-do objects inside of it. Each todo object should have its properties as in the example below:
```mdx
const tasks = [
  {
    description: 'Clean my room',
    completed: false,
    index: 1,
  },
 {
    description: 'Cook some banana',
    completed: false,
    index: 2,
  },
];
```

- [ ] Next, you should create a function that looks over the tasks array and render them to the screen.
- [ ] When the page loads, it suppose to render all the tasks to the screen.
- [ ] Also, please consider adding the clear all completed button at the bottom of the todos just as in the original design because you'll have to write a logic in your next milestones to clear all the completed todos. That'll come in later 👍  

You will be doing all the above requirements inside of the `index.js` file 👍